src/components/Fields/TextField.story.js

```diff
@@ -400,3 +400,17 @@ _NB: if you add a prefix and an icon, only the prefix will be displayed_
         </div>,
     ),
 );
+
+stories.add(
+    'Autocomplete',
+    storyWrapper(
+        `
+\`autoComplete\` prop allows you to control whether the browser should autocomplete input value
+        `,
```

Comment:
<img width="223" alt="screenshot 2019-01-15 at 13 24 05" src="https://user-images.githubusercontent.com/30833983/51183350-d8b86600-18c8-11e9-9015-19d325a508a1.png">

looks like this `,` appears in storybook next to the field

Reply:
oops :)